supreme_court_predictions/processing/token_aggregation.py

```diff
@@ -0,0 +1,237 @@
+"""
+This file provides tokens per case.
+"""
+import pickle
+
+import pandas as pd
+
+from supreme_court_predictions.util.contants import ENCODING_UTF_8
+from supreme_court_predictions.util.files import get_full_pathway
+
+
+class TokenAggregations:
+    def __init__(self, save_data=True):
+        self.all_tokens = None
+        self.advocate_tokens = None
+        self.adversary_tokens = None
+        self.judge_tokens = None
+
+        self.save_data = save_data
+
+        # Get local directory
+        self.local_path = get_full_pathway(
+            "/supreme_court_predictions/data/clean_convokit/"
+        )
```

Comment:
I feel like I should add the `supreme_court_predictions bit` on to this function as we use it every time.